Formula/s/svt-av1.rb

```diff
@@ -21,7 +21,14 @@ class SvtAv1 < Formula
   depends_on "nasm" => :build
 
   def install
-    system "cmake", "-S", ".", "-B", "build", "-DCMAKE_INSTALL_RPATH=#{rpath}", *std_cmake_args
+    extra_cmake_args = %W[-DCMAKE_INSTALL_RPATH=#{rpath}]
+
+    # Explicitly disable I8MM extension: upstream build script attemps to detect
+    # CPU features via compiler flags, which are stripped by brew's compiler shim.
+    # Not all Apple Silicon architectures (namely M1) support the I8MM extension.
+    extra_cmake_args << "-DENABLE_NEON_I8MM=OFF"
```

Comment:
Is this can be `ON` for linux?

Reply:
That's fair, we can at least avoid setting it explicitly in that case (if aarch64 Linux becomes officially supported, we can revisit this once decided on which minimum instruction set/extensions).